js/src/oai_wrappers/non_proxy_wrappers.ts

```diff
@@ -0,0 +1,139 @@
+import { Span, startSpan } from "../logger";
```

Comment:
This is purely a cut-and-paste from `oai.ts`, except for changing the wrapper function name.